applications/asset_tracker_v2/src/modules/cloud_module.c

```diff
@@ -812,6 +812,25 @@ static void on_sub_state_cloud_connected(struct cloud_msg_data *msg)
 				true);
 	}
 
+	if (IS_EVENT(msg, data, DATA_EVT_WIFI_ACCESS_POINTS_DATA_SEND)) {
+#if defined(CONFIG_LOCATION_METHOD_WIFI)
+		if (IS_ENABLED(CONFIG_LWM2M_INTEGRATION)) {
+			err = cloud_wrap_wifi_access_points_send(NULL, 0, true, 0);
+			if (err) {
+				LOG_ERR("cloud_wrap_wifi_access_points_send, err: %d", err);
+			}
+
+			return;
+		}
+#endif
+
+		add_qos_message(msg->module.data.data.buffer.buf,
```

Comment:
I guess this needs to be inside the idef

Reply:
True. Or actually it should probably be around entire:
`if (IS_EVENT(msg, data, DATA_EVT_WIFI_ACCESS_POINTS_DATA_SEND)) {`.